src/templates/eye_upper_slim.csv

```diff
@@ -0,0 +1,20 @@
+ID,subset,label
+ID,AI oboInOwl:inSubset,
+CL:0000210,http://purl.obolibrary.org/obo/cl#eye_upper_slim,photoreceptor cell

```

Comment:
Consider if '[eye photoreceptor cell](http://purl.obolibrary.org/obo/CL_0000287)' is more appropriate.